src/components/Reviews.jsx

```diff
@@ -1,9 +1,37 @@
+import { useState, useEffect } from "react"
+
 function Reviews () {
+
+    const [reviews2, setReviews] = useState([])
+
+    useEffect(() => {
+        fetch("https://db-reviews.onrender.com/api/reviews")
+            .then(result => {
+            return result.json()
+            })
+            .then(({ reviews })  => {
+                console.log(reviews)
```

Comment:
While this is happening the user should be given visual feedback that the data is still loading.